css/main.css

```diff
@@ -143,92 +143,95 @@ h2#Sverde{
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divServicios ul li{
+div#div-servicios ul li {
     margin-bottom: 10px;
 }
 
-div#divServicios #serviceCon{
+div#div-servicios #service-con {
     display: flex;
     flex-wrap: wrap;
     padding: 180px;
 }
 
-div.service{
+div.service {
     width: 175px;
     display: inline-block;
     margin-left: 50px;
     margin: 10px;
```

Comment:
Let's remove the `margin-left` property and use the `margin` shorthand

Reply:
done